### PR TITLE
gifdiff: add page

### PR DESCRIPTION
**Pull request by CairnThePerson**
Just adding this to go along with `gifsicle`.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.93

### CODE REVIEW

File: pages/common/gifdiff.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# gifdiff
+
+> `diff` for GIFs.
```
Comment (kbdharun) [inline pages/common/gifdiff.md:3]: Welcome to tldr @CairnThePerson,  I think we can reword this line a bit as mentioned [here](https://github.com/kohler/gifsicle).
```suggestion
> Compare two GIFs for identical visual appearance. 
```

Comment (CairnThePerson) [reply pages/common/gifdiff.md:3]: That's fair!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks @CairnThePerson

File: pages/common/gifdiff.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# gifdiff
+
+> Compare two GIFs for identical visual appearance.
```
Comment (kbdharun) [inline pages/common/gifdiff.md:3]:
```suggestion
> Compare two GIFs for identical visual appearance.
```
Remove this whitespace

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**